arangod/Replication2/ReplicatedState/LeaderStateManager.tpp

```diff
@@ -53,7 +53,11 @@ void LeaderStateManager<S>::run() noexcept {
           data.updateInternalState(LeaderInternalState::kIngestingExistingLog);
           auto mux = Multiplexer::construct(self->logLeader);
           mux->digestAvailableEntries();
+#ifdef _MSC_VER                                   // circumventing bug in msvc
+          data.stream = mux->getStreamById<1>();  // TODO fix stream id
```

Comment:
please create a BTS for this todo.

Reply:
why?

Reply:
That makes sense, I assumed @dothebart was referring to the TODO note `// TODO fix stream id`.

Reply:
please don't create at ticket. There is already a ticket in the CINFRA board that would remove this piece of code completely. https://arangodb.atlassian.net/browse/CINFRA-160

Reply:
ok, thanks @maierlars - I just wanted this not to be forgotten.